packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -106,6 +107,7 @@ function GridCell(props: GridCellProps) {
     onKeyDown,
     onDragEnter,
     onDragOver,
+    touchRippleRef,
```

Comment:
It was passed to a `div` and caused warning, right?

Reply:
My bad, that is not needed - reverted. There was an issue with this but it was for the checkbox